src/machi.proto

```diff
@@ -424,6 +426,20 @@ message Mpb_LL_ReadChunkResp {
     repeated Mpb_ChunkPos trimmed = 3;
 }
 
+// Low level API: trim_chunk()
+
+message Mpb_LL_TrimChunkReq {
+    required Mpb_EpochID epoch_id = 1;
+    required string file = 2;
+    required uint64 offset = 3;
+    required uint32 size = 4;
+    optional uint32 trigger_gc = 5 [default=1];
+}
```

Comment:
I'm curious about the use of this new option ... and why its default is 1/true.  Is this intended for debugging/development use only, or do you believe that it should remain as part of the public API?

One of the JIRA scheduled tasks is to add a background management process for checksum "scrub" and also GC space reclamation.  It isn't scheduled yet, but it needs to be added before the first major product release.